quic/s2n-quic-events/events/common.rs

```diff
@@ -93,6 +93,8 @@ impl IntoEvent<bool> for &crate::transport::parameters::MigrationSupport {
     }
 }
 
+#[builder_derive(derive(Copy))]
+#[derive(Debug, Clone, Copy)]
```

Comment:
We don't want copy here so we reserve the right to add non-copyable values. They all implement clone though so just use that.

Reply:
yep, thats a really good point. https://github.com/aws/s2n-quic/pull/1422/commits/470046fe985853930f2fb08788702e7fe1fb851b

Reply:
Since the builder is not a public api, we can add Copy to those items and remove it later if we want. Added the Copy trait just to the builder items in https://github.com/aws/s2n-quic/pull/1422/commits/e814bb6a629316c648797d4f858bd86ea2a708e5